config/prow/cluster/monitoring/kube-state-metrics_servicemonitors.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   labels:
     app.kubernetes.io/name: kube-state-metrics
     app.kubernetes.io/version: 1.9.7
+    app: kube-state-metrics
```

Comment:
Why are both this and https://github.com/kubernetes/test-infra/pull/21375 needed?

Reply:
Oh nevermind, I see the revert now: https://github.com/kubernetes/test-infra/pull/21378